src/components/accession2/DetailPanel.tsx

```diff
@@ -143,7 +143,7 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
               {strings.PLANT_AND_SITE}
             </Grid>
             <Grid item xs={gridRightSide}>
-              {`${strings.COLLECTED_FROM} ${accession.plantsCollectedFromMax}-${accession.plantsCollectedFromMin} ${strings.WILD_PLANT}`}
+              {`${strings.COLLECTED_FROM} ${accession.plantsCollectedFrom} ${strings.WILD_PLANT}`}
```

Comment:
guessing WILD_PLANT will be changed to read from the record?

Reply:
Yes, we can update it now if it's in the api

Reply:
yes it is the collectionSource, i will update.